### PR TITLE
Un-cavemans some strain buy information, and changes every strain flavor text from 40k/doom references

### PR DESCRIPTION
**Pull request by InsaneRed**
# About the pull request

changes up the stupid "talls, auto spitters etc" inside OOC strain buying information.
Re-writes all strainflavor text.

# Explain why it's good for the game

Cooler(imo) strain flavor texts are better when not all of them are warhammer references.
its weird as hell to use IC in OOC information like "talls, auto spitters". (its also stupid as hell that they speak like this anyway, or whoever intrdouced this)

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
spellcheck: New flavor text and better strain information
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by InsaneRed** [comment]
chill bro i dont want you

File: code/modules/mob/living/carbon/xenomorph/strains/castes/boiler/trapper.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-	flavor_description = "Hsss, I love the smell of burnin' tallhost flesh in the mornin'."
+	description = "You trade your ability to bombard, lance, and dump your acid in order to gain some speed and the ability to create acid explosions and restrain enemies within them. With your longer-range vision, set up traps that immobilize your opponents and place acid mines which deal damage to enemies and barricades and reduce the cooldown of your trap deployment for every enemy hit. Finally, hit enemies with your Acid Shotgun ability which adds a stack of insight to empower the next trap you place once you reach a maximum of ten insight. A point-blank shot or a shot on a stunned target will instantly apply ten stacks."
+	flavor_description = "The battlefield my canvas, this one, my painter. Melt them where they stand."
```
Comment (Git-Nivrak) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/boiler/trapper.dm:4]:
```suggestion
	flavor_description = "The battlefield is my canvas, this one, my painter. Melt them where they stand."
```

File: code/modules/mob/living/carbon/xenomorph/strains/castes/crusher/charger.dm [hunk 2]
```diff
@@ -14,8 +14,8 @@
...
-	flavor_description = "We're just getting started. Nothing stops this train. Nothing."
+	description = "In exchange for your shield, a little bit of your armor and damage, your slowdown resist from turrets, your influence under frenzy pheromones, your stomp no longer knocking down talls, and your ability to lock your direction, you gain a considerable amount of health, some speed, your stomp does extra damage when stomping over a grounded tall, and your charge is now manually-controlled and momentum-based; the further you go, the more damage and speed you will gain until you achieve maximum momentum, indicated by your roar. In addition, your armor is now directional, being the toughest on the front, weaker on the sides, and weakest from the back. In return, you gain an ability to tumble to pass through enemies and avoid enemy fire, and an ability to forcefully move enemies via ramming into them."
+	flavor_description = "Nothing stops this hive. This one will become both the immovable and the unstoppable object."
```
Comment (Git-Nivrak) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/crusher/charger.dm:18]:
```suggestion
	flavor_description = "Nothing stops this hive. This one will become both the immovable object and the unstoppable force."
```

File: code/modules/mob/living/carbon/xenomorph/strains/castes/crusher/charger.dm [hunk 3]
```diff
@@ -14,8 +14,8 @@
...
-	flavor_description = "We're just getting started. Nothing stops this train. Nothing."
+	description = "In exchange for your shield, a little bit of your armor and damage, your slowdown resist from turrets, your influence under frenzy pheromones, your stomp no longer knocking down talls, and your ability to lock your direction, you gain a considerable amount of health, some speed, your stomp does extra damage when stomping over a grounded tall, and your charge is now manually-controlled and momentum-based; the further you go, the more damage and speed you will gain until you achieve maximum momentum, indicated by your roar. In addition, your armor is now directional, being the toughest on the front, weaker on the sides, and weakest from the back. In return, you gain an ability to tumble to pass through enemies and avoid enemy fire, and an ability to forcefully move enemies via ramming into them."
```
Comment (Git-Nivrak) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/crusher/charger.dm:17]:
```suggestion
	description = "In exchange for your shield, a little bit of your armor and damage, your slowdown resist from sentries, your influence under frenzy pheromones, your stomp no longer knocking down talls, and your ability to lock your direction, you gain a considerable amount of health, some speed, your stomp does extra damage when stomping over a grounded tall, and your charge is now manually-controlled and momentum-based; the further you go, the more damage and speed you will gain until you achieve maximum momentum, indicated by your roar. In addition, your armor is now directional, being the toughest on the front, weaker on the sides, and weakest from the back. In return, you gain an ability to tumble to pass through enemies and avoid enemy fire, and an ability to forcefully move enemies via ramming into them."
```

File: code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/dancer.dm [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-	flavor_description = "Demonstrate to the talls what 'there is beauty in death' truly symbolizes, then dance upon their graves!"
+	description = "You lose all of your acid-based abilities and a small amount of your armor in exchange for increased movement speed, evasion, and unparalleled agility that gives you an ability to move even more quickly, dodge bullets, and phase through enemies and allies alike. By slashing enemies, you temporarily increase your movement speed and you also you apply a tag that changes how your two new tail abilities function. By tagging enemies, you will make Impale hit twice instead of once and make Tail Trip knock enemies down instead of stunning them."
+	flavor_description = "A performance fitting for a Queen, This one will become my instrument of death."
```
Comment (Git-Nivrak) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/dancer.dm:5]:
```suggestion
	flavor_description = "A performance fit for a Queen, This one will become my instrument of death."
```

File: code/modules/mob/living/carbon/xenomorph/strains/castes/ravager/berserker.dm [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
-	flavor_description = "They shall be my finest warriors. They will rend and tear, crush and butcher, and maim and rage until every tallhost falls."
+	description = "You lose your empower, charge, and scissor cut, decrease your health, and sacrifice a bit of your influence under frenzy pheromones to increase your movement speed, slightly increase your armor, and gain a new set of abilities that make you a terrifying melee monster. By slashing, you heal yourself and gain a stack of rage that increases your armor, movement speed, attack speed, and your heals per slash, to a maximum of six rage. Use your new Appehend ability to increase your movement speed and apply a slow on the next target you slash and use your Clothesline ability to fling your target to heal yourself, even more-so if you have a rage stack that will be used up. Finally, use your Eviscerate to unleash a devastating windmill attack that heals you for every enemy you hit after an immobilizing wind-up."
+	flavor_description = "Unbridled fury fills this one, You will become an extension of my rage."
```
Comment (Git-Nivrak) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/ravager/berserker.dm:4]:
```suggestion
	flavor_description = "Unbridled fury fills this one, This one will become an extension of my rage."
```
consistency

**Comment by Birdtalon** [comment]
I'm all in favour of rewriting out the stuff about talls, spitters etc but the little flavour text references to Thanos and stuff I'm not particularly in favour of replacing, but then again I'm a little inactive at the moment so won't stand in the way.

**Comment by InsaneRed** [comment]
i just dont think marvel references / 40k references have place (they're also TOO cringy)

File: code/modules/mob/living/carbon/xenomorph/strains/castes/ravager/berserker.dm (hunk 5 above)
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/ravager/berserker.dm:4]: Either full stop, or fix the capitalization.

File: code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/dancer.dm [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
-	flavor_description = "Demonstrate to the talls what 'there is beauty in death' truly symbolizes, then dance upon their graves!"
+	description = "You lose all of your acid-based abilities and a small amount of your armor in exchange for increased movement speed, evasion, and unparalleled agility that gives you an ability to move even more quickly, dodge bullets, and phase through enemies and allies alike. By slashing enemies, you temporarily increase your movement speed and you also you apply a tag that changes how your two new tail abilities function. By tagging enemies, you will make Impale hit twice instead of once and make Tail Trip knock enemies down instead of stunning them."
+	flavor_description = "A performance fit for a Queen, This one will become my instrument of death."
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/dancer.dm:5]:
```suggestion
	flavor_description = "A performance fit for a Queen, this one will become my instrument of death."
```

File: code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/warden.dm [hunk 7]
```diff
@@ -1,8 +1,8 @@
...
-	flavor_description = "Only in death does your sisters' service to the Queen end. They will be untouched by plague or disease; no sickness will blight them."
+	description = "You trade your acid ball, acid spray, dash, and a small bit of your slash damage and speed to become an effective medic. You gain the ability to emit strong pheromones, an ability that retrieves endangered, knocked-down or resting allies and pulls them to your location, and you gain an internal hitpoint pool that fills with every slash against your enemies, which can be spent to aid your allies and yourself by healing them or curing their ailments."
+	flavor_description = "This one will deny her sisters death untill they earn it. Fight or be forgotten."
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/warden.dm:5]:
```suggestion
	flavor_description = "This one will deny her sisters' deaths until they earn it. Fight or be forgotten."
```

**Review by Drulikar — APPROVED** [review]

**Pull request merged**